iphone/iphone/Titanium_Prefix.pch

```diff
@@ -11,10 +11,10 @@
 #include "ThirdpartyNS.h"
 #import <AvailabilityInternal.h>
 
-#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 110000
-#define IS_XCODE_9 true
```

Comment:
Since you removed the Xcode 9 check this means we will bump the minimum supported Xcode version to 9, right?

Reply:
Yes.